helm-base-versions/send-json-to-cosmos.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
It would be good to have some test coverage for this script ideally I think

Reply:
checking if its easy to move over to any sdk